### PR TITLE
fix(thumbnail) Thumbnail redesign tests fix

### PR DESCRIPTION
**Pull request by robertpin**

### CODE REVIEW

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

File: src/test/java/org/jitsi/meet/test/util/MeetUIUtils.java [hunk 1]
```diff
@@ -1004,20 +999,7 @@ public static void waitForNinjaIcon(WebDriver driver)
...
-    }
-
-    /**
-     * Waits for dominant speaker icon to appear in remote video of a participant.
-     * @param driver the <tt>WebDriver</tt> instance where the check will be
-     * performed.
-     * @param endpointId the endpoint ID of the participant whose dominant speaker icon
-     * status will be checked.
-     */
-    public static void waitForDominantspeaker(WebDriver driver, String endpointId)
```
Comment (damencho) [inline src/test/java/org/jitsi/meet/test/util/MeetUIUtils.java:1017]: Why this is removed without an alternative? Seems to me this changes the logic of the tests

Comment (robertpin) [reply src/test/java/org/jitsi/meet/test/util/MeetUIUtils.java:1017]: My bad. Fixed it :D

**Comment by robertpin** [comment]
Jenkins test this please

**Review by damencho — APPROVED** [review]

**Review by yanas — APPROVED** [review]

**Pull request merged**